.dev/docker/windows/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# escape=`
```

Comment:
What about using escape as \, just like for linux based docker files? Will it interfere with the path? (And I heard that Windows can take forward slash for paths now)

Reply:
I wouldn't really like to try. Time is already at a minimum 😄